docs/get-started/data-collection.md

```diff
@@ -195,7 +239,43 @@ result = (
 towhee.plot(query.to_list(), result.to_list())
 ```
 
+## Prepare LMDB files for Pytorch Training
+
+```python
+>>> DataCollection.from_glob('path_train/*.jpg') \
```

Comment:
Is there a DC API docs? I think need to explain what is from_glob, from_zip, read_text and load_image etc.

Reply:
add data collection API doc with sphinx